src/screens/PlayScreen/Play.tsx

```diff
@@ -1,9 +1,41 @@
-import { Image, Text, View } from 'react-native';
+import { Image, Text, View, Pressable } from 'react-native';
+import { Audio } from 'expo-av';
+import React from 'react';
 import Icon from '../../../assets/icons';
 import Colors from '../../styles/Colors';
 import styles from './styles';
+import { RootStackScreenProps } from '../../types/navigation';
+
+// TODO: Use toggleAudio when it is necessary, may be fixed in later PRs.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+async function toggleAudio(
```

Comment:
Remove this function for now -- you can find it in the commit history in this PR if you need it later.